packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -109,4 +109,35 @@ cbp-structured-list {
   div[role=list]:only-child > *:first-child {
     border-top: var(--cbp-border-size-md) solid var(--cbp-structured-list-border-color);
   }
+
+  /** Selectable */
+  cbp-structured-list-item cbp-checkbox{
+    display: inline-flex;
+
+    input[type=checkbox]{
```

Comment:
This doesn't feel right - you shouldn't be overriding CSS on these elements in this component. Leverage flex and use a gap, ideally. Even if you had to add a margin, it could be done on the component rather than the nested input checkbox.

Reply:
removed this style block and applied gap to the &[selectable] cbp-structured-list-item{} selector below this